distro/configs/odoo/initializer_config/auth_providers/auth.oauth.provider.csv

```diff
@@ -1,3 +1,3 @@
 "id","sequence","name","client_id","client_secret","enabled","flow","token_map","auth_endpoint","validation_endpoint","scope","body","data_endpoint","token_endpoint","jwks_uri","end_session_endpoint","skip_logout_confirmation","autologin"
 "init.provider_openerp","10","Odoo.com Accounts","","",false,"id_token_code","","https://accounts.odoo.com/oauth2/auth","https://accounts.odoo.com/oauth2/tokeninfo","userinfo","Log in with Odoo.com","","","","",false,false
-"init.provider_keycloak","10","Keycloak","odoo","${ODOO_CLIENT_SECRET}",true,"id_token_code","preferred_username:user_id","${KEYCLOAK_URL}/realms/ozone/protocol/openid-connect/auth","${KEYCLOAK_URL}/realms/ozone/protocol/openid-connect/userinfo",profile email openid,"Login with Single Sign-On","","${KEYCLOAK_URL}/realms/ozone/protocol/openid-connect/token","${KEYCLOAK_URL}/realms/ozone/protocol/openid-connect/certs","${KEYCLOAK_URL}/realms/ozone/protocol/openid-connect/logout",true,true
+"init.provider_keycloak","10","Keycloak","odoo","${ODOO_CLIENT_SECRET}",true,"id_token_code","preferred_username:user_id","${KEYCLOAK_URL}/realms/ozone/protocol/openid-connect/auth","${KEYCLOAK_INTERNAL_HOST_URL}/realms/ozone/protocol/openid-connect/userinfo",profile email openid,"Login with Single Sign-On","","${KEYCLOAK_INTERNAL_HOST_URL}/realms/ozone/protocol/openid-connect/token","${KEYCLOAK_INTERNAL_HOST_URL}/realms/ozone/protocol/openid-connect/certs","${KEYCLOAK_INTERNAL_HOST_URL}/realms/ozone/protocol/openid-connect/logout",true,true
```

Comment:
How/where is this new `KEYCLOAK_INTERNAL_HOST_URL` var set?

Reply:
It's internal Docker url for Keycloak. It always been there see https://github.com/ozone-his/ozone-docker-compose/blob/main/.env#L167